lib/llhttp/parser.js

```diff
@@ -57,11 +53,6 @@ const wasm_on_url = (p, at, length) => {
 }
 
 const wasm_on_status = (p, at, length) => {
-  const ret = currentParser[kTrackHeader](length)
-  if (ret !== 0) {
-    return ret
-  }
-  currentParser[kStatusMessage] = cstr(at, length)
   return 0
```

Comment:
if this function is empty, is it still needed?

Reply:
Yes, otherwise the WASM fails

Reply:
@dnlup we might want to make this optional there, it will save a few cycle.

Reply:
Yes, all the functions exported in https://github.com/nodejs/undici/blob/main/deps/llhttp/src/api.c#L29 must be provided in the initialization.